src/sonic-bgpcfgd/bgpcfgd/manager_static_route.py

```diff
@@ -0,0 +1,173 @@
+import traceback
+from .log import log_crit, log_err, log_debug
+from .manager import Manager
+from .template import TemplateFabric
+import socket
+
```

Comment:
Did we test save & restore for STATIC_ROUTE table?

Reply:
Do you mean saving and restoring items in STATIC_ROUTE table in config_db? There should be a config command to apply changes to config_db. I think the test for that is not going to be covered by this PR. There would be unit tests for the config command. And we are planning to have sonic-mgmt tests to test the static route feature as a whole.

Reply:
ok